paddleslim/quant/helper.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2019  PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import matplotlib
+matplotlib.use('Agg')
+import logging
+import numpy as np
+from matplotlib.backends.backend_pdf import PdfPages
+import matplotlib.pyplot as plt
+import os
+
+import paddle
+import paddle.fluid as fluid
+
+from ..common import get_logger
+_logger = get_logger(__name__, level=logging.INFO)
+
+
+def draw_var_distribution_hist(program,
```

Comment:
Please move this function to `paddleslim.common` package and rename it `paddleslim.common.pdf`?

Reply:
done